src/PlusImageProcessing/Testing/vtkPlusTransverseProcessEnhancerTest.cxx

```diff
@@ -6,7 +6,7 @@ See License.txt for details.
 
 /*!
 \file vtkPlusTransverseProcessEnhancerTest.cxx
-\brief TODO Give a one-liner explanation as to the basics of this test
+This is a program meant to test vtkPlusTransverseProcessEnhancer.cxx from the command line.

```

Comment:
Perhaps be a bit more specific like "This program loads an XYZ sequence from file and generates some ABC data..." or something